src/merkle/mmr/partial.rs

```diff
@@ -0,0 +1,81 @@
+//! A partial view of the MMR, useful to maintain an update proof for a single value.
+use super::{super::Vec, full::nodes_in_forest, Rpo256, Word};
+
+/// A partial view of the MMR.
+///
+/// Data structure used to maintain the proof of a single value up-to-date. Only the data relevant
+/// to `value` is stored, meaning the path from the leaf to the root, and the peaks in the MMR that
```

Comment:
When we say `path from the leaf to the root`, can we specify in the docs if we refer to a peak or absolute root (i.e. the root of the peaks)?

Reply:
Updated it to `path from the leaf to the peak`.

Reply:
And to clarify, this implementation can not have the `hash(peaks)`. That would imply the partial view would need to be updated with every message.